.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+<!-- This template is for adding new packages. If you are suggesting a different change, feel free to delete it -->
+
+## Add package to ecosystem
+
+To add a package to the scverse ecosystem registry, please add the package metadata to `packages.yml`.
+Additionally, please answer the following questions. After submitting the PR, your request will be reviewed
+by the scverse core team (usually within a week).
+
+Name of the tool: XXXXXXX
+Short description: XXXXXX
+
+**Where is the source code published (please provide link)**
+
+**Is the package interoperable with the appropriate scverse data structures (i.e. AnnData for single-cell datasets,
+MuData for multi-omics data)? Please state in a few sentences how the package uses the data structures.**
+
+**Where is the documentation available? Please provide the link**
+
+**Does the package have automated tests and does it use continuous integration? Please provide links to the respective
+code sections.**
+
+**Does the package provide versioned releases?**
+
+**Is the package installable from at least one standard registry (e.g. PyPI or conda-forge)**
+
+**Under what software licence is the tool published? We only accept OSI-approved licenses**
+
+**Where can users see what level of support is available?**
+
+**Does the package provide tutorials (aka vignettes) to help new users getting started quickly? Please provide links**
+
+**Does the package use the scverse cookiecutter template?**
+
+## Statements
+
+-   [ ] I am the author or maintainer of the tool and agree on listing the package on the scverse website
```

Comment:
Do we want something like this? If we do, we should also get approval for all packages we add ourselves in the beginning. 

On conda-forge they require the package maintainer to agree. This is usually solved by pinging them on github.